experimenter/experimenter/targeting/constants.py

```diff
@@ -2270,6 +2270,22 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+DEFAULT_AUTOFILL_ADDRESSES_SUPPORTED = NimbusTargetingConfig(
+    name="Users with Default or Non-'On' Setting for Address Autofill",
+    slug="default_autofill_addresses_supported",
+    description=(
+        "Targets users who have left the 'extensions.formautofill.addresses.supported' "
+        "preference at its default value or set it to something other than 'on'."
+    ),
+    targeting=(
+        "!('extensions.formautofill.addresses.supported'|preferenceIsUserSet) || "
+        "'extensions.formautofill.addresses.supported'|preferenceValue != 'on'"
+    ),
+    desktop_telemetry="",
+    sticky_required=False,
+    is_first_run_required=False,
+    application_choice_names=(Application.DESKTOP.name,),
+)
```

Comment:
I think you just need to add a blank line after this to make the formatter happy. You can also run `make code_format` but that will take longer :P